qa-heal.planx-pla.net/dashboard/Public/documentation/index.html

```diff
@@ -913,36 +1235,72 @@ <h4 id="CurrentlyAvailableNotebooks" class="section-title">Currently available N
+                <ul>
+                    <li>
+                        <span style="font-weight: bold;">Some notebooks require linked access to FAIR enabled repositories</span>, 
+                        as described <a href="#LinkingAccessTo">here</a>. 
+                        <br>
+                        The <a href="#CurrentlyAvailableNotebooks">previous section</a>
+                        lists if the notebook needs linking access to FAIR enabled repositories.
+                    </li>
+                </ul>
+                <ul>
+                    <li>
+                        Code in the notebooks is editable, and users can import additional datasets and extend their analysis. 
+                    </li>
+                </ul>
+
+                
+
+            
+            <h4 id="DemoHEALNotebookBACPAC" class="section-title">Demo - How to find data and work on the Tutorial Notebooks</h4>
+                <p>
+                    Below we describe the steps to export data from the Discovery Page tutorial Jupyter notebooks
+                    for one example tutorial notebook 
+                    HEAL_Workspace_Demo_BACPAC.ipynb (“BACPAC synthetic data analysis”)
+                </p>
+                <blockquote>
+                    This example notebook does not require linking access to a FAIR repository.
+                    Please see the table in the <a href="#CurrentlyAvailableNotebooks">previous section</a> if 
+                    the notebook of interest requires linking access.
+                </blockquote>
                 <p>
-                    Code in the notebooks is editable, and users can import additional datasets and extend their analysis. 
-                    In order to work with the tutorial Jupyter Notebooks in interactive mode, the data used by the tutorial 
-                    notebook must be exported to a workspace from the Discovery page. <br><br>
-                    Below we describe the steps to do so for two tutorial Jupyter notebooks. <br>
                     <span style="font-weight: bold;">Users must be logged in to follow the steps below.</span>
-                    Note, that some notebooks above require linked access to FAIR enabled repositories, as described
-                    <a href="#LinkingAccessTo">here</a>.
                 </p>
-            
-            <h5 id="HEALNotebookBACPAC" class="section-title">a) HEAL_Workspace_Demo_BACPAC.ipynb (“BACPAC synthetic data analysis”)</h5>
+
                 <ol>
                     <li>
-                        Find and select the study by the project number (1U24AR076730-01) on the <a href="http://healdata.org/discovery">Discovery Page</a>. Project numbers and datasets are given in the previous section.<br><br>
+                        Find and select the study by the project number (in this example: 1U24AR076730-01) 
+                        on the <a href="http://healdata.org/discovery">Discovery Page</a>. 
```

Comment:
Very minor detail, but all links should be `https` by default. Just saves a basic http -> https redirect for modern browsers.